src/richText.js

```diff
@@ -232,6 +232,15 @@ define([
                 editor.fire("saveSnapshot");
                 return wrapper;
             },
+            focus: function() {
+                if (editor.status === "ready") {
+                    editor.focus();
+                } else {
+                    editor.on('instanceReady', function () {
```

Comment:
Wonder if it would make sense to set a flag indicating that the editor will be focused when it's ready and do not create another event handler if it is set? That would prevent creating a pile of `instanceReady` even handlers if `focus()` is called many times before the instance is ready.